### PR TITLE
Fixed a few typos in the docs

### PR DESCRIPTION
**Pull request by pitmonticone**
Just fixed a few typos in the documentation. 

Thank you very much for the development and maintenance of such a useful GP package!

### CODE REVIEW

File: examples/getting_started/script.jl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # # Getting Started
 #
-# Here we document how to do some basic stuff, including learning and inference in kernel parameters, with Stheno.jl. To do this, we that makes use of a variety of packages from the Julia ecosystem. In particular, we'll make use of
+# Here we document how to do some basic stuff, including learning and inference in kernel parameters, with Stheno.jl. To do this, it adopts a variety of packages from the Julia ecosystem. In particular, we'll make use of
```
Comment (willtebbutt) [inline examples/getting_started/script.jl:3]:
```suggestion
# Here we document how to do some basic stuff, including learning and inference in kernel parameters, with Stheno.jl. To do this, we make use of a variety of packages from the Julia ecosystem:
```

**Review by willtebbutt — COMMENTED** [review]
Thanks for opening this and the kind words -- it's good to know someone is reading the docs! I've just left one suggestion. Happy to merge once we've agreed on that.

**Comment by pitmonticone** [comment]
Ok, done. 

Thanks again :)

**Comment by willtebbutt** [comment]
Will merge when CI passes.

**Pull request merged**